src/mockup-db/categoryDbo.json

```diff
@@ -0,0 +1,58 @@
+[
+    {
+        "id": "1",
+        "name": "Drawing",
+        "CategoryGroupId": "1",
```

Comment:
Do the `CategoryGroupId` and `id` fields should be as string values?

Reply:
Yes, Id fields are string type all in DB as far as we can see from the backend repository and the coming date